packages/web/src/hooks/useVersion.ts

```diff
@@ -30,7 +30,8 @@ const useRemoteVersion = () => {
       if (!packageJson) {
         return false;
       } else {
-        return version !== packageJson.data.version;
+        // Compare only major versions
+        return version.split('.')[0] !== packageJson.data.version.split('.')[0];
```

Comment:
Since patch and minor version updates will become more frequent, update notifications will be changed to major versions only